narratives/ncov_sit-rep_2020-03-20.md

```diff
@@ -0,0 +1,2453 @@
+
+
+<!-- There is NO right-side text -->
+
+
+<!-- ############ SLIDE BREAK ############# -->
+<!-- This is left-side text 5 -->
+# [Sequence data can cross-check travel histories](https://nextstrain.org/ncov/2020-03-19?legend=closed&&c=division_exposure&label=clade:A1a&d=tree,map)
+
+Identifying where a given case was infected is important for understanding which areas are experiencing local transmission versus primarily travel-associated cases. Sequence data can help us differentiate between these scenarios.
+<br><br>
+Here, the tree is colored by the travel history of a case (when known). Mousing over the blue Canadian sequence in the middle, we see that they have a reported travel history to Europe. This infection groups with other European sequences, showing that this case almost certainly is a travel-related infection.
+
+<!-- ############ SLIDE BREAK ############# -->
+<!-- This is left-side text 6 -->
+# [Sequence data can cross-check travel histories](https://nextstrain.org/ncov/2020-03-19?legend=closed&branchLabel=clade&c=division_exposure&d=tree,map&p=full)
+
+Zooming out, we can see that this isn't always so consistent.
+For example, mousing over the bright green Canadian sequence in the top right (near clade A7), we see that this case has a travel history to Iran.
+However, this sequence doesn't group anywhere near the rest of the cases with Iranian travel history! This makes it much less clear where this person was actually infected, despite the travel history.
+
+<!-- There is no right side text -->
+
+<!-- ############ SLIDE BREAK ############# -->
+
+<!-- This is left-side text 7 -->
+# [The virus has been introduced to nearly every country around the world](https://nextstrain.org/ncov/2020-03-19?legend=closed&branchLabel=clade&c=country&d=tree,map&p=grid)
+
+On the tree, we see lots of this kind of intermingling of samples from different countries.
+This indicates that this kind of inevitable human movement has already introduced the virus to many parts of the globe.
+<br><br>
+In fact, the WHO reports confirmed cases in 159 out of 195 total countries in the world.
+<br><br>
+However, it's important to keep in mind that not all introductions result in outbreaks.
+
+<!-- There is no right side text -->
+
+<!-- ############ SLIDE BREAK ############# -->
+<!-- This is left-side text 8 -->
+# [Epidemics become evident ~4-8 weeks after introduction](https://nextstrain.org/ncov/2020-03-19?legend=closed&branchLabel=clade&c=num_date&d=tree&f_division=Washington&label=clade:B1&p=full)
+
+It may seem as though COVID-19 has suddenly exploded -- for many people, this outbreak may have gone from an abstract concern in other parts of the world to something that severely impacts our daily lives, seemingly overnight.
```

Comment:
Consider making "for many people..." a brand new sentence. Also, "our" should be "their".